src/renderer/tr_shader.cpp

```diff
@@ -244,6 +254,81 @@ void R_RemapShader(const char *shaderName, const char *newShaderName, const char
 	}
 }
 
+void R_RemapShaderAdvanced(const char *shaderName, const char *newShaderName, const char *timeOffset, shaderRemapLightmapType_t lightmapMode, shaderRemapStyleType_t styleMode) {
```

Comment:
Would be better to have float timeOffset argument instead of parsing it this deep down.

Reply:
Adjusted in fe717366f617d4645db0540ee714e971345b81d3 (using int instead of float).